### ISSUE TITLE
α and β

### ISSUE THREAD

**Issue by xuexuegoudan** [huaidanquede/PrimeK-Net#3]
Could you please clarify what α and β represent in the paper and what their specific values are? I noticed they weren't explicitly mentioned in the text.

**Comment by huaidanquede** [comment]
Thank you for pointing that out. α and β are learnable parameters; they do not have fixed values and will vary during the training process.
&nbsp;




---Original---
From: ***@***.***&gt;
Date: Thu, Nov 6, 2025 10:26 AM
To: ***@***.***&gt;;
Cc: ***@***.***&gt;;
Subject: [huaidanquede/PrimeK-Net] α and β (Issue #3)


xuexuegoudan created an issue (huaidanquede/PrimeK-Net#3)
 
Could you please clarify what α and β represent in the paper and what their specific values are? I noticed they weren't explicitly mentioned in the text.

**Comment by xuexuegoudan** [comment]
Thank you for your reply. I would like to ask about your equipment and running time. Is it reasonable that one training epoch takes me one hour on a single RTX 3090? Also, how do you choose the optimal model? Do you select the weights with the smallest PESQ on the validation set?

**Comment by huaidanquede** [comment]
I trained on a single RTX4060ti 16G, your training time is normal. In most cases, higher PESQ represents better result.



---Original---
From: ***@***.***&gt;
Date: Fri, Nov 7, 2025 21:39 PM
To: ***@***.***&gt;;
Cc: "Zizhen ***@***.******@***.***&gt;;
Subject: Re: [huaidanquede/PrimeK-Net] α and β (Issue #3)


xuexuegoudan left a comment (huaidanquede/PrimeK-Net#3)
 
Thank you for your reply. I would like to ask about your equipment and running time. Is it reasonable that one training epoch takes me one hour on a single RTX 3090? Also, how do you choose the optimal model? Do you select the weights with the smallest PESQ on the validation set?

**Comment by xuexuegoudan** [comment]
你好，我还有个问题，我看到在训练阶段中每个batch都计算pesq，这可能是导致训练时间长的原因，请问如果我改成只在验证阶段计算pesq,这是合理的吗，会对模型效果有很大影响吗？
